src/Stryker.Core/Stryker.Core/Baseline/Utils/BaselineMutantHelper.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+using Stryker.Core.Mutants;
+using Stryker.Core.Reporters.Json;
+
+namespace Stryker.Core.Baseline.Utils
+{
+    public class BaselineMutantHelper : IBaselineMutantHelper
```

Comment:
not too thrilled on the naming of this class, any suggestions? @Mobrockers

Reply:
These are the helper methods we use to get the mutants in the projecft-under-test that match a given mutant from the json baseline.  It also allows to get the original source code of a json mutant. These methods were originally part of the DiffMutantFilter.

Reply:
@Mobrockers keep it this way?

Reply:
It's not great but it's not terrible. I don't currently have a better suggestion.